app/Jobs/SetWikiLogo.php

```diff
@@ -66,7 +66,7 @@ public function handle(): void
         $logosDir = Wiki::getLogosDirectory($wiki->id);
 
         // Upload the local image to the cloud storage
-        $storage->putFileAs($logosDir, new File($this->logoPath), "raw.png");
+        $storage->putFileAs($logosDir, new File($this->logoPath), "raw.png", ['visibilty' => 'public']);
```

Comment:
We could also set the visibility on global level, i.e. in the driver's configuration. Not sure which approach is preferrable?